app/src/main/java/com/rafraph/pnineyHalachaHashalem/textMain.java

```diff
@@ -1,2790 +1,2788 @@
+			bNext_page.setImageDrawable(resources.getDrawable(R.drawable.ic_action_down));
+			bPrevious_page.setImageDrawable(resources.getDrawable(R.drawable.ic_action_up));
+			if(cameFromSearch == true)
+			{
+				bFindNext.setImageDrawable(resources.getDrawable(R.drawable.ic_action_down));
+				bFindPrevious.setImageDrawable(resources.getDrawable(R.drawable.ic_action_up));
+			}
+		}
+
+		MenuInflater DownMenu = getMenuInflater();
+		if(MyLanguage == ENGLISH)
+			DownMenu.inflate(R.menu.down_menu_english, menu);
+		else if(MyLanguage == RUSSIAN)
+			DownMenu.inflate(R.menu.down_menu_russian, menu);
+		else if(MyLanguage == SPANISH)
+			DownMenu.inflate(R.menu.down_menu_spanish, menu);
+		else if(MyLanguage == FRENCH)
+			DownMenu.inflate(R.menu.down_menu_french, menu);
+		else
+			DownMenu.inflate(R.menu.down_menu, menu);
+		return true;
+	}//onCreateOptionsMenu
+
+	public void onBackPressed()
+	{
+		if(fullScreenFlag == 1)
+		{
+			fullScreenFlag = 0;
+			getSupportActionBar().show();
+			lnrOptions.setVisibility(View.VISIBLE);
+		}
+		else
+		{
+			super.onBackPressed();
+		}
+	}
+
+	public void onConfigurationChanged(Configuration newConfig)
+	{
+		super.onConfigurationChanged(newConfig);
+		// Checks the orientation of the screen
+		if (newConfig.orientation == Configuration.ORIENTATION_LANDSCAPE)
+		{
+			rotate=2;
+		}
+		else if (newConfig.orientation == Configuration.ORIENTATION_PORTRAIT)
+		{
+			rotate=1;
+		}
+	}
+
+	@Override
+	public boolean dispatchKeyEvent(KeyEvent event)
+	{
+		int keyCode = event.getKeyCode();
+		int keyAction = event.getAction();
+		switch (keyCode)
+		{
+			case KeyEvent.KEYCODE_VOLUME_UP:
+				if(keyAction == KeyEvent.ACTION_UP)
+				{
+					webview.pageUp(false);
+				}
+				return true;
+			case KeyEvent.KEYCODE_VOLUME_DOWN:
+				if(keyAction == KeyEvent.ACTION_UP)
+				{
+					webview.pageDown(false);
+				}
+				return true;
+			default:
+				return super.dispatchKeyEvent(event);
+		}
+	}
+
+	int scrollSpeed=1;
+	private Handler mHandler=new Handler();
+	public Runnable mScrollDown = new Runnable()
+	{
+		public void run()
+		{
+			if(scrollSpeed == 0) // in case of note opened
+			{
+				mHandler.postDelayed(this, 200);
+			}
+			else if(scrollSpeed == -1) // in case that "stop" pressed
+			{
+				webview.scrollBy(0, 0);
+			}
+			else
+			{
+				webview.scrollBy(0, 1);
+				mHandler.postDelayed(this, 200/scrollSpeed);
+			}
+		}
+	};
+
+	@SuppressWarnings("deprecation")
+	@Override
+	public void onClick(View view)
+	{
+		String currentChapter;
+		// TODO Auto-generated method stub
+		switch(view.getId())
+		{
+			case R.id.ibChapters:
+				findHeaders();
+				showPopupMenu(view);
+				break;
+
+			case R.id.switchModes:
+
+
+
+
+
+				final Context context = this;
+				dialogModes = new Dialog(context);
+				dialogModes.setTitle("בחר מצב משתמש");
+				dialogModes.setContentView(R.layout.activity_switch_modes);
+				Button hearandreadBtn = (Button) dialogModes.findViewById(R.id.hearAndReadBtn);
```

Comment:
Please use camel case. In this case change hearandreadBtn to hearAndReadBtn

Reply:
תוקן  לא שמתי לב